liquibase-core/src/main/java/liquibase/diff/output/StandardObjectChangeFilter.java

```diff
@@ -31,37 +43,39 @@ protected void parseFilter(String filter) {
         for (String subfilter : filter.split("\\s*,\\s*")) {
             String[] split = subfilter.split(":");
```

Comment:
It is better 'standard' name for  String.split() result - "tokens" (or even "splitted" is more descriptive - it hinting rightaway what is that). It just suggestion, no insisting

Reply:
I stole this part of code from liquibase-core OSS, I would prefer to keep it as is